connectors/cache/fallback.go

```diff
@@ -49,12 +49,31 @@ type rangeQuery struct {
 	Limit      int
 }
 
+// Options returns a function that's being used for connector initialization
+type Options func(*Connector) error
+
+// WithSkipWriteInvalidateEntities provides the option for client to set the entites
+func WithSkipWriteInvalidateEntities(entities ...dosa.DomainObject) Options {
+	return func(c *Connector) error {
+		c.skipWriteInvalidateEntitiesMap = createSkipWriteCacheInvalidateSet(entities)
+		return nil
+	}
+}
+
 // NewConnector creates a fallback cache connector
-func NewConnector(origin, fallback dosa.Connector, scope metrics.Scope, entities ...dosa.DomainObject) *Connector {
-	return newConnector(origin, fallback, scope, encoding.NewGobEncoder(), entities...)
+func NewConnector(origin, fallback dosa.Connector, scope metrics.Scope, entities []dosa.DomainObject, options ...Options) *Connector {
```

Comment:
i don't think this function makes much sense if you're passing in Option that can only take a fallback Connector as an argument, but this function passes in a dosa.Connector interface.